wiki/fr/fr-Installation-for-Windows.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: wiki
+title: "Installation sous Windows"
+lang: "fr"
+permalink: "/wiki/Installation-for-Windows"
+---
+
+# Installation sous Windows
+
+Assurez-vous d'avoir déjà lu la page [Premier pas](Getting-Started) avant de commencer.
```

Comment:
"_avant de commencer_" there is no such a thing in the English version. Please remove.

Reply:
I'd be ok with that since it doesn't change the meaning of the text. 

We have to discuss to which degree we need to keep close to the English version (should be part of the contribution discussion).

On the one hand we must ensure that the translation doesn't sound like a french version of an English text (it should sound like a native french text), but on the other hand we need to make sure that every user knows the same.

Reply:
> I'd be ok with that since it doesn't change the meaning of the text.

I agree with @trebmuh for this, it is not in the english, and no need (that's me… with my need to insist on "before start"), but in fact it's not necessary. « Assurez vous d'avoir lu… » is enougth.